aurt/api.py

```diff
@@ -90,6 +90,14 @@ def calibrate_validate(model_path, data_path, gravity, calibration_data_relative
     params = rc.calibrate()
     output_pred = rc.predict(rc.robot_data_validation, gravity, params)
 
+    # ************************************************************************************
```

Comment:
Are these comments needed or can they be removed?

Reply:
Those comments can be safely deleted. In future releases of AURT, one might want to actually test the FMU and for that purpose this code may again be relevant. I guess the comments should just be deleted now, it's not like it would take months to write again :-)